frontend/packages/kubevirt-plugin/src/components/create-vm-wizard/create-vm-wizard.tsx

```diff
@@ -391,6 +404,8 @@ export const CreateVMWizardPageComponent: React.FC<CreateVMWizardPageComponentPr
     );
   }
 
+  const storageClassConfigMap = useStorageClassConfigMapWrapped();
```

Comment:
this hook should load the correct config map for us. IMO it shouldn't be a problem to do this only once per wizard as the map should not change dynamically. And even if  it does change it probably makes sense to have a consistent behaviour for a single instance of a wizard